src/main/app/forms/form.ts

```diff
@@ -47,19 +47,22 @@ export class Form<Model> {
 
   @ValidateNested()
   model: Model
+  rawData: object
   errors: FormValidationError[]
 
   /**
    * @param model - a object used to fill the form
+   * @param rawData - a raw data used to create model instance
    * @param errors - an array of error objects
    */
-  constructor (model: Model, errors: ValidationError[] = []) {
+  constructor (model: Model, rawData: object = undefined, errors: ValidationError[] = []) {
```

Comment:
If you added the new parameter after existing ones then you wouldn't need to assign default value to `rawData`, it would be `undefined` by default if not provided. An additional example of such approach is that you're sure you don't break any existing calls. Please change this.